### PR TITLE
update qiime.org landing page

### PR DESCRIPTION
**Pull request by gregcaporaso**
I've seen several instances lately of people being referred to qiime.org when they should be referred to qiime2.org. This update provides a simple page that (manually) redirects the user to qiime2.org, but gives them an option to still access the old qiime.org page. 

Preview of new index page:

Screenshot 2022-12-19 at 11 35 59 AM

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,90 +1,36 @@
...
-					<p>You can find the <a class="reference external" href="http://www.nature.com/nmeth/journal/vaop/ncurrent/full/nmeth.f.303.html">QIIME paper here</a>, and the data presented in this paper can be found
-						<a class="reference external" href="ftp://ftp.microbio.me/qiime/QIIME_NM_2010.tgz">here</a>.
-					<br><br>
-					</p>
+	<header>
+		<img src="home_static/images/qiime2.png"/>
+	</header>
+	
+  	<div id="content">
+	<h1>QIIME has been succeeded by <a href="https://qiime2.org">QIIME 2</a>&nbsp; &nbsp;&nbsp;</h1>
```
Comment (wasade) [inline index.html]: Should this be `QIIME&trade`? Or does that not matter?

**Comment by wasade** [comment]
Just a small question otherwise :+1:

File: index.html (hunk 1 above)
Comment (gregcaporaso) [reply index.html]: _Probably_ doesn't matter, but I added anyway. Thanks @wasade. 

Do you mind squashing and merging?

Comment (gregcaporaso) [reply index.html]: Screenshot 2022-12-19 at 11 43 56 AM

**Comment by gregcaporaso** [comment]
Thanks @wasade!

**Pull request merged**